### PR TITLE
Adds menu page

### PR DESCRIPTION
**Pull request by zFl4wless**
The menu page contains just two buttons:

- A "start" button, which will start the actual trainer (doesn't work yet)
- A "quit" button, which will just close the app

### CODE REVIEW

File: src/main/java/com/github/zfl4wless/ui/pages/MenuPage.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import com.github.zfl4wless.Main;
+import com.github.zfl4wless.ui.pages.listeners.QuitButtonActionListener;
+import com.github.zfl4wless.ui.pages.listeners.StartButtonActionListener;
+
+import javax.swing.*;
+
+public class MenuPage {
+    private static final JPanel page = Main.getApp().getPageHandler().createPage();
+
+    public static JPanel view() {
```
Comment (zBinFinn) [inline src/main/java/com/github/zfl4wless/ui/pages/MenuPage.java:12]:
```suggestion
    public static JPanel render() {
```
would suggest a refactor to better describe the method, idk

File: src/main/java/com/github/zfl4wless/ui/pages/MenuPage.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        JButton startButton = new JButton("Start");
+        startButton.addActionListener(new StartButtonActionListener());
+        JButton quitButton = new JButton("Quit");
+        quitButton.addActionListener(new QuitButtonActionListener());
```
Comment (zBinFinn) [inline src/main/java/com/github/zfl4wless/ui/pages/MenuPage.java:16]: You should probably make a custom button class

**Review by zBinFinn — CHANGES_REQUESTED** [review]
### **Remove all of those unnecessary this.'s please.**

**Comment by zFl4wless** [comment]
I would like to leave the "this."'s because I am really used to them. SORRY :(

**Review by zBinFinn — APPROVED** [review]
LGTM

**Pull request merged**